math/gr1c/Portfile

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           muniversal 1.0
+PortGroup           github 1.0
+
+github.setup        tulip-control gr1c 0.13.2 v
 
-name                gr1c
-version             0.9.0
 categories          math devel
 platforms           darwin
-maintainers         caltech.edu:SCL openmaintainer
+maintainers         cds.caltech.edu:slivingston openmaintainer
```

Comment:
Sorry, missed this, your Github handle should be here as well

Reply:
My fault: I should have generalized your previous comment. Thanks for reviewing this!